### PR TITLE
KAA-952 Remove KAA_TRACE_IN and KAA_TRACE_OUT from tests

### PR DESCRIPTION
**Pull request by Dubland**
/cc @rasendubi @forGGe

### CODE REVIEW

**Comment by forGGe** [comment]
Must be rebased on newest true_metal

File: client/client-multi/client-c/test/kaa_tcp_channel/test_kaa_tcp_channel_bootstrap.c [hunk 1]
```diff
@@ -205,8 +201,7 @@ void test_create_kaa_tcp_channel(void **state)
 void test_set_access_point_full_success_bootstrap(void **state)
 {
     (void)state;
-    KAA_TRACE_IN(logger);
-
+
```
Comment (rasendubi) [inline client/client-multi/client-c/test/kaa_tcp_channel/test_kaa_tcp_channel_bootstrap.c]: Remove trailing spaces.

**Comment by rasendubi** [comment]
LGTM

**Comment by forGGe** [comment]
Looks good to me

**Pull request merged**